modules/Microsoft.App/containerApps/.test/min/dependencies.bicep

```diff
@@ -0,0 +1,42 @@
+@description('Required. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Log Analytics Workspace to create.')
+param logAnalyticsWorkspaceName string
+
+@description('Required. The name of the Managed Environment for Container Apps to create.')
+param managedEnvironmentName string
+
+resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2020-03-01-preview' = {
+  name: logAnalyticsWorkspaceName
+  location: location
+  properties: any({
+    retentionInDays: 30
+    features: {
+      searchVersion: 1
+    }
+    sku: {
+      name: 'PerGB2018'
+    }
+  })
+}
+
+resource managedEnvironment 'Microsoft.App/managedEnvironments@2022-10-01' = {
+  name: managedEnvironmentName
+  location: location
+  sku: {
+    name: 'Consumption'
+  }
+  properties: {
+    appLogsConfiguration: {
+      destination: 'log-analytics'
+      logAnalyticsConfiguration: {
```

Comment:
A LAW is optional for the managed environment.

Reply:
removed